index.html

```diff
@@ -0,0 +1,262 @@
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 426.4 82.5" width="160" height="40">
+                        <style>.st0{fill:#d3d3d3}</style><title>facebook</title><g id="_Group_"><path id="_Path_" class="st0" d="M33.4 
+                        13c-5 0-6.5 2.2-6.5 7.1v8.1h13.4L39 41.4H26.9v39.9H10.8V41.4H0V28.3h10.8v-7.9C10.8 7 16.2 0 31.1 0c3.1 0 6.3.2 
+                        9.4.6V13"/><path id="_Compound_Path_" class="st0" d="M41.4 53.1c0-14.8 7-26 21.8-26 8 0 12.9 4.1 15.3 
+                        9.3v-8.1h15.4v53.1H78.4v-8c-2.2 5.1-7.3 9.1-15.3 9.1-14.7 0-21.8-11.1-21.8-26m16.2.2c0 7.9 2.9 13.2 10.4 13.2 6.6 0 
+                        9.9-4.8 9.9-12.4v-5.4c0-7.6-3.4-12.4-9.9-12.4-7.5 0-10.4 5.2-10.4 13.2v3.8z"/><path id="_Path_2" class="st0" d="M127.3 
+                        27.1c6.2 0 12.2 1.3 15.4 3.6l-3.6 11.4c-3.3-1.6-7-2.4-10.7-2.4-8.7 0-12.5 5-12.5 13.6v3.1c0 8.6 3.8 13.6 12.5 13.6 3.7 
+                        0 7.4-.9 10.7-2.5l3.6 11.4c-3.2 2.2-9.1 3.6-15.4 3.6-18.9 0-27.4-10.1-27.4-26.4v-2.5c0-16.3 8.5-26.5 27.4-26.5"/>
+                        <path id="_Compound_Path_2" class="st0" d="M144.8 56.8v-4.7c0-15.1 8.6-25 26.1-25 16.5 0 23.8 10 23.8 24.8v8.5h-33.8c.3 
+                        7.3 3.6 10.5 12.5 10.5 5.8 0 11.6-1.1 17.1-3.2l2.9 11c-4.2 2.2-12.9 3.9-20.6 
+                        3.9-20.4-.1-28-10.2-28-25.8m16.1-7h19.4v-1.3c0-5.8-2.3-10.4-9.4-10.4-7.2 0-10 4.6-10 11.7"/><path id="_Compound_Path_3" 
+                        class="st0" d="M254.3 56.5c0 14.8-7.1 26-21.9 26-8 0-13.6-4-15.8-9.1v8h-15.2V1.6L217.5.2v35.5c2.3-4.7 7.5-8.5 15-8.5 
+                        14.7 0 21.9 11.1 21.9 26m-16.2-.4c0-7.5-2.9-13.1-10.6-13.1-6.6 0-10.1 4.7-10.1 12.3v5.6c0 7.6 3.6 12.3 10.1 12.3 7.7 0 
+                        10.6-5.6 10.6-13.1v-4z"/><path id="_Compound_Path_4" class="st0" d="M259.6 56.1v-2.7c0-15.3 8.7-26.3 26.4-26.3s26.4 11 26.4 
+                        26.3v2.7c0 15.3-8.7 26.3-26.4 26.3s-26.4-10.9-26.4-26.3m36.8-3.8c0-7-2.9-12.6-10.4-12.6s-10.4 5.6-10.4 12.6v5c0 7 2.9 12.6 
+                        10.4 12.6s10.4-5.6 10.4-12.6v-5z"/><path id="_Compound_Path_5" class="st0" d="M317.9 56.1v-2.7c0-15.3 8.7-26.3 
+                        26.4-26.3s26.4 11 26.4 26.3v2.7c0 15.3-8.7 26.3-26.4 26.3s-26.4-10.9-26.4-26.3m36.8-3.8c0-7-2.9-12.6-10.4-12.6s-10.4 
+                        5.6-10.4 12.6v5c0 7 2.9 12.6 10.4 12.6s10.4-5.6 10.4-12.6v-5z"/><path id="_Path_3" class="st0" d="M392.8 
+                        53.6l15.9-25.4h17l-16.6 26.3 17.3 26.9h-17.1l-16.5-26v26h-16V1.6l16-1.5"/></g></svg>
+                </div>
+            </section>
+            <section class="footer">
+                <img class="low-logo" src="./images/logo.png" alt="logo">
+                <div class="copyright lato">
+                    Emily Robertson, 2022. This is a dummy website for educational purposes only, based on a template by Cindy Shin.
+                </div> 
+            </section>
+        </footer>
```

Comment:
- [ ] Please make sure to include the footer in the mobile screen version of your app
_I know that the design frame doesn't show it. However, it doesn't make sense to remove the footer of the page in the mobile screen version right?_ 😕